app/sidero-controller-manager/main.go

```diff
@@ -216,6 +217,7 @@ func main() {
 		os.Exit(1)
 	}
 
+	setupWebhooks(mgr)
```

Comment:
I still don't get how this would work. In https://github.com/siderolabs/sidero/pull/794 I used a dedicated controller for the webhooks because the reconcilers above this line are actively using the conversion webhooks. 

To my understanding this will cause Sidero to be unable to start when any of the CRD's change like proposed in https://github.com/siderolabs/sidero/pull/735, because in that case the reconcilers will try to convert them using the conversion webhooks defined here (but at that point, they are not yet available).

Reply:
@lion7 I'm going to take a closer look today at your PR and do more testing for the conversion

Reply:
👍🏻 that would be great, in that case I'll wait with rebasing https://github.com/siderolabs/sidero/pull/735 so it won't accidentally break (I have a working deployment of it running on my cluster).